bert_reranker/scripts/normalize_data.py

```diff
@@ -21,9 +21,11 @@ def remove_html_tags(s):
     return re.sub(clean, '', s)
 
 
-def remove_standard_punctuation(s):
-    punctuation = '¿!"#$%&\'()*+,-./:;<=>?@[\\]^`{|}~'  # We are leaving out _ for the _URL_ token
-    return s.translate(str.maketrans(punctuation, ' ' * len(punctuation)))
+NON_STANDARD_PUNCTUATION_RE = re.compile('¿')
```

Comment:
why define this outside the function?

Reply:
I wanted to be in a well-evident variable...
you are right that it may be top of the file, but here is closer to its usage.

anyway, I think it's not supeor important because soon enough we will make this configurable..